pkgs/flox-cli/default.nix

```diff
@@ -126,9 +126,6 @@
     // lib.optionalAttrs hostPlatform.isDarwin {
       NIX_COREFOUNDATION_RPATH = "${darwin.CF}/Library/Frameworks";
       PATH_LOCALE = "${darwin.locale}/share/locale";
-    }
-    // lib.optionalAttrs hostPlatform.isLinux {
-      LOCALE_ARCHIVE = "${glibcLocalesUtf8}/lib/locale/locale-archive";
```

Comment:
@ysndr Are you sure we should remove this?

Reply:
I got my answer here :) https://github.com/flox/flox/actions/runs/7170558816/job/19523563824?pr=592

I think we should revert this change.

Reply:
Yeah I was too eager when undoing the first change